concrete/elements/navigation/application_menu.php

```diff
@@ -61,14 +48,13 @@
                         </li>
                         <li>
                           <a href="#" data-launch-sub-panel-url="<?=URL::to('/ccm/system/panels/dashboard/load_menu'); ?>"
-                          data-load-menu="<?=$myAccount; ?>">
+                          data-load-menu="192">
```

Comment:
@aembler I need to get the collection ID of /account/edit_profile in order to not hard code this ID here.  Is there a way to get a page by its url?

Reply:
Yes, `Concrete\Core\Page\Page::getByPath('/account/edit_profile')` should get you a page object you can use.

Reply:
thanks @aembler ... I just figured that one out (duur) and was about to delete this comment. :)